sessions/recap-project-6-NEW/activity-planner/README.md

```diff
@@ -0,0 +1,31 @@
+# Activity Planner
```

Comment:
I would rename the folder here just recap-project-6, the students dont need to know that this one is the new project and after half a year the "new" is a bit awkward.